scripts/check-license-compliance.js

```diff
@@ -0,0 +1,103 @@
+#! /usr/bin/env node
+'use strict';
+/* eslint no-console: [ "error", { allow: [ "info", "warn", "error", "table" ] } ] */
+const { init } = require('license-checker'),
+      process = require('process'),
+      util = require('util'),
+      initLicenseChecker = util.promisify(init),
+      packageDirectory = require('pkg-dir');
+
+const ALLOWED_LICENSES = [
+   'MIT',
+   'ISC',
+   'Apache-2.0',
+   '(MIT OR Apache-2.0)',
+   'Public Domain',
+   'CC-BY-3.0',
+];
+
+(async () => {
+   const projectDirectoryPath = await packageDirectory(),
+         projectConfigPath = `${projectDirectoryPath}/.license-checker.cjs`,
```

Comment:
Any particular reason this is .cjs and not just .js?

Reply:
Fixed